include/platform/radio.h

```diff
@@ -273,6 +278,13 @@ ThreadError otPlatRadioHandleTransmitDone(bool *aFramePending);
 int8_t otPlatRadioGetNoiseFloor(void);
 
 /**
+ * Get the radio capabilities.
+ *
+ * @returns The radio capabilities.
+ */
+uint32_t otPlatRadioGetCaps(void);
```

Comment:
Add more doxygen.  For example indicate that this is returning a bit vector.  Also mention kRadioCapsAckTimeout and make sure it's linked so that you can click on it and see the documentation for that flag.